.github/workflows/modular-docs-publish.yml

```diff
@@ -19,6 +19,8 @@ jobs:
               branch: modular-docs-rhosak
             - product: RHOSR
               branch: modular-docs-rhosr
+            - product: RHODA
```

Comment:
That would be typically to push ascidocs splitter/access.redhat.com documentation.
Since we have only yaml files output of that job will be always empty.

Reply:
```suggestion
            # - product: RHODA
```

Reply:
@wtrocki : Since we are not publishing AsciiDoc via the Customer Portal using this mechanism, would commenting out or remove lines 22, 23 allow the job to compile successfully?